### PR TITLE
Release 1.15

### PR DESCRIPTION
**Pull request by SMYALTAMASH**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,27 +1,10 @@
...
-RUN pwd
-
-#Running MLWB
-RUN pip3 install -r /home/ML-Workbench/requirements.txt
+FROM python:3.6.5 as buildingml
+WORKDIR /home/
+WORKDIR /home/ml-workbench/
+RUN apt update && git clone https://github.com/project-sunbird/sunbird-ml-workbench -b release-1.15 .
+RUN bash -x build.sh \
+        && pip install bin/daggit-0.5.0.tar.gz \
```
Comment (rjshrjndrn) [inline Dockerfile]: please remove this line till 8.
And copy the artifacts from the container, to a final container, from which we'll run the MLWB.

File: devops/ansible/install-ml-workbench.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+- hosts: ml-workbench
+  gather_facts: false
+#  pre_tasks:
```
Comment (rjshrjndrn) [inline devops/ansible/install-ml-workbench.yml]: please remove all commented lines

File: devops/ansible/hosts [hunk 3]
```diff
@@ -0,0 +1,2 @@
+[ml-workbench]
+IP ansible_ssh_user=test ansible_ssh_key_file=~/.ssh/keyFile
```
Comment (rjshrjndrn) [inline devops/ansible/hosts]: don't use ansible_ssh_user in the line itself

File: devops/ansible/roles/install-ml-workbench/defaults/main.yml [hunk 4]
```diff
@@ -0,0 +1,48 @@
+virtualenv_dir: "/home/{{ansible_ssh_user}}/mlwb_venv"
```
Comment (rjshrjndrn) [inline devops/ansible/roles/install-ml-workbench/defaults/main.yml]: please remove all ansible for provisioning MLWB

File: Dockerfile (hunk 1 above)
Comment (SMYALTAMASH) [reply Dockerfile]: we will do it as a multistage container with build in one container and installer dependencies, ml-workbench in another container

**Pull request closed without merging**